### PR TITLE
to suppress email submission completely with token config

### PR DESCRIPTION
**Pull request by volodymyrss**
I will generate the token elsewhere, but I need this suppression for larger analysises I run.

### CODE REVIEW

File: tests/test_server_basic.py [hunk 1]
```diff
@@ -473,6 +473,10 @@ def test_email_oda_api(dispatcher_live_fixture, dispatcher_local_mail_server):
         **default_token_payload
     }
     encoded_token = jwt.encode(token_payload, secret_key, algorithm='HS256')
+    
+    if isinstance(encoded_token, bytes):
```
Comment (burnout87) [inline tests/test_server_basic.py:477]: Was this case happening ?

File: cdci_data_analysis/analysis/tokenHelper.py [hunk 2]
```diff
@@ -42,11 +44,35 @@ def get_token_user_sending_submitted_interval_email(decoded_token):
...
\ No newline at end of file
+        return jwt.decode(token, secret_key, algorithms=[default_algorithm])
+
+def update_token(token, secret_key, payload_mutation: FunctionType):
+    token_payload = jwt.decode(token, secret_key, algorithms=[default_algorithm])    
+
+    # update here
+    # tem=300000
+
+    token_payload = payload_mutation(token_payload)
```
Comment (burnout87) [inline cdci_data_analysis/analysis/tokenHelper.py:66]: Is the goal of the `payload_mutation` to just change the content of the token payload? And sometimes also to return it?

**Review by burnout87 — APPROVED** [review]
Just a couple of quetions

File: tests/test_server_basic.py (hunk 1 above)
Comment (volodymyrss) [reply tests/test_server_basic.py:477]: yes, that's why I added it. It depends on pyjwt version I guess.

File: cdci_data_analysis/analysis/tokenHelper.py (hunk 2 above)
Comment (volodymyrss) [reply cdci_data_analysis/analysis/tokenHelper.py:66]: actually, this part it's not finished. And I did also some token inspection/modification in oda_api:
https://github.com/oda-hub/oda_api/pull/56/files which starts to duplicate this. So I remove, and think later where to put it better.

**Pull request merged**